### PR TITLE
Block translation

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Block/Social/TwitterEmbedTweetBlockService.php [hunk 1]
```diff
@@ -93,27 +93,19 @@ public function buildEditForm(FormMapper $form, BlockInterface $block)
     {
         $form->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
-                array('tweet', 'textarea', array('required' => true, 'help_block' => 'Tweet id, URL or content. If you put the HTML content directly, other options won\'t be handled.')),
-                array('maxwidth', 'integer', array('required' => false, 'help_block' => 'Must be contained between 250 and 550')),
-                array('hide_media', 'checkbox', array('required' => false, 'help_block' => 'Hide media contained in tweet?')),
+                array('tweet', 'textarea', array('required'       => true, 'help_block' => 'Tweet id, URL or content. If you put the HTML content directly, other options won\'t be handled.')),
```
Comment (OskarStark) [inline Block/Social/TwitterEmbedTweetBlockService.php]: this is wrong, you should update your php-cs-fixer version

**Comment by OskarStark** [comment]
awesome work @core23

File: Block/Social/TwitterEmbedTweetBlockService.php (hunk 1 above)
Comment (core23) [inline Block/Social/TwitterEmbedTweetBlockService.php]: Used the the latest version. Maybe we should reformat some stuff? @Soullivaneuh @rande 

```php

        $formMapper->add('settings', 'sonata_type_immutable_array', array(
            'keys' => array(
                array('url', 'url', array(
                    'required' => false
                )),
                array('image', 'text', array(
                    'required' => false
                )),
                array('description', 'text', array(
                    'required' => false
                )),
                array('size', 'integer', array(
                    'required' => false
                )),
                array('shape', 'choice', array(
                    'required' => false,
                    'choices'  => array(
                        'rectangular' => 'rectangular',
                        'round'       => 'round',
                    ),
                )),
            ),
        ));

```

Comment (OskarStark) [inline Block/Social/TwitterEmbedTweetBlockService.php]: really? this bug was fixed, did you run ``composer global update`` ?

**Pull request merged**